docs/testing-components/unit-testing/index.md

```diff
@@ -205,25 +190,27 @@ const Text = (props) => {
 
 //Example A - Bad
 test('Should pass prop to component', () => {
-	const Text = shallow(
-	    <Text content='sample' />
+	render(
```

Comment:
please modify the Text component above so the < p > cand receive the rest of the props

Reply:
Done